framework/src/base/Moose.C

```diff
@@ -228,6 +228,7 @@ addActionTypes(Syntax & syntax)
   registerTask("check_integrity", true);
   registerTask("resolve_optional_materials", true);
   registerTask("check_integrity_early", true);
+  registerTask("check_integrity_early_physics", false);
```

Comment:
if you register any action to check_integrity_early, it gets created even if it s not in the input file
that's because of the ", true)" for required just above

Reply:
🤦